FileShareService.DesktopClient.Core/Jobs/NewBatchJob.cs

```diff
@@ -52,11 +53,62 @@ public void Validate(JToken jsonToken)
             }
 
             //Check for files
-            if (jsonToken.SelectToken("actionParams.files") != null && 
+            if (jsonToken.SelectToken("actionParams.files") != null &&
                 jsonToken.SelectToken("actionParams.files").Type != JTokenType.Array)
             {
                 ErrorMessages.Add($"Invalid file object.");
             }
+
+            //Check for file attributes
+
+            foreach (JToken fileObj in jsonToken.SelectToken("actionParams.files"))
+            {
+                JToken? fileAttributeToken = fileObj.SelectToken("attributes");
+                var searchPath = fileObj.SelectToken("searchPath");
+
+                if (fileAttributeToken == null) continue;
+                else
+                {
+                    if (fileAttributeToken?.Type != JTokenType.Array)
+                    {
+                        ErrorMessages.Add("Invalid file attribute. searchPath:" + searchPath);
+                        continue;
+                    }
+                    if (!fileAttributeToken.HasValues) continue;
+                    else
```

Comment:
this else is redundant, we can remove this and reduce indentation

Reply:
updated the same, Thanks